code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -116,6 +118,12 @@
 		evolve()
 		return
 
+/// This proc is used to determine if a mouse should chew on a wire or not. Returns TRUE if it should, FALSE if it shouldn't.
+/mob/living/simple_animal/mouse/proc/should_chew()
```

Comment:
Remove `always_chew` and just have this `return prob(15)`, and override it in the subtype